STY: improve QC of unprocessed data (#414)
TheAxonLab/hcph-sops: docs/data-management/mriqc.md

    ```
??? warning "We are not yet registering the *MRIQC* derivatives dataset as a sub-dataset."

    To avoid complication at run time, the *MRIQC* derivatives dataset will be added as a sub-dataset to the unprocessed dataset only after *MRIQC* run is completed. 

>     As such, note that we are **NOT** using the `-d` flag

Reviewer: This is presented as some sort of feature, but it is actually a bug and is unrelated to the warning:
```suggestion
```

        │   ├── sub-001_ses-001_task-qct_dir-AP_bold.html
        │   ├── sub-001_ses-001_task-rest_dir-AP_bold.html
    ```
    

>     <a id="mriqc-failed"></a>

Reviewer: Sweet :)

        - [ ] Address the issue (e.g., out-of-memory when running a container) and re-run *MRIQC*.
            Proceed as in the previous case by finding documentation on the *MRIQC* repository or NeuroStars.

- [ ] Visualize the first mosaic (background mosaic) and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#view-of-the-background-of-the-anatomical-image).

!!! warning "Running through the visual report does not have to be executed in a fixed order"

>     While we describe the run through the report as linear, often inspecting other reportlets can help make a decision about the exclusion criteria related to the reportlet at hand. As such, we encourage you to jump back and forth between visualizations as much as needed.

Reviewer: ```suggestion
    !!! tip "Do not hesitate to jump back and forth through sections while screening the visual report"
```

- [ ] Scroll down to the zoom in the zoomed-in brain mosaic and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#zoomed-in-mosaic-view-of-the-brain).

> - [ ] Verify that no error is reported in the section "About > Errors". If there is, follow the procedure described in "*MRIQC* failed to produce all the expected visual reports".

Reviewer: Semantic line breaks
```suggestion
- [ ] Verify that no errors are reported in subsection "Errors" of section "About".
    If there are, follow the procedure described in "*MRIQC* failed to produce all the expected visual reports".
```

Reviewer: Also, it would be nice to add an anchor right before or inside the admonition block and convert that `"*MRIQC* failed to produce all the expected visual reports"` into a link.

#### QCT

!!! note "We are checking QCT fMRI first and in principle not excluding QCT scans"

>     Except if the image is [extremely distorted](qaqc-criteria-unprocessed.md#task-fmri-exclusion-criteria), we are not excluding QCT scans because we will leverage those images to evaluate the quality of fMRI scans and derived constructs throughout the whole analysis pipeline. We are however going through the *MRIQC* reports to train our eye, anticipate issues that might be affecting RSfMRI and flag the corresponding sessions.

Reviewer: ```suggestion
    Except if the image is [extremely distorted](qaqc-criteria-unprocessed.md#task-fmri-exclusion-criteria), we are not excluding QCT scans because we will leverage those images to evaluate the quality of fMRI scans and derived constructs throughout the whole analysis pipeline.
    It is important to screen and rate the QCT runs as they may serve as a proxy for the quality of the other fMRI tasks in a session and display issues that might be affecting the RSfMRI and BHT.
```

- [ ] Open each *MRIQC* report on a current Web Browser (*Google Chrome* is preferred).
- [ ] Repeat the following two steps for each echo:
    - [ ] Visualize the first mosaic (standard-deviation) and search for [artifacts](qaqc-criteria-unprocessed.md#standard-deviation-of-signal-through-time)

>     - [ ] Scroll down to the carpetplot and search for [artifacts](qaqc-criteria-unprocessed.md#carpetplot-and-nuisance-signals).

Reviewer: ```suggestion
    - [ ] Search for [artifacts](qaqc-criteria-unprocessed.md#carpetplot-and-nuisance-signals) in the carpetplot visualization.
```

> - [ ] Once you went through all the echo-wise visualization of the base report, scroll down to the "Extended echo-wise reports" section, inspect the background view and search for [artifacts](qaqc-criteria-unprocessed.md#view-of-the-background-of-the-voxel-wise-average-of-the-bold-timeseries).

Reviewer: ```suggestion
- [ ] Visualize all echo-wise visualizations of the base report
- [ ] Inspect the background view and search for [artifacts](qaqc-criteria-unprocessed.md#view-of-the-background-of-the-voxel-wise-average-of-the-bold-timeseries) in the "Extended echo-wise reports" section.
```

Please describe what we are looking for in the echo-wise visualizations.

> - [ ] Scroll down to the average BOLD mosaic and search for [artifacts](qaqc-criteria-unprocessed.md#average-signal-through-time)

Reviewer: ```suggestion
- [ ] Search for [artifacts](qaqc-criteria-unprocessed.md#average-signal-through-time) in the average BOLD mosaic visualization.
```

- [ ] Inspect the zoomed-in view of the average BOLD mosaic as well and search for the same [artifacts](qaqc-criteria-unprocessed.md#average-signal-through-time).

> 

Reviewer: ```suggestion
```

> #### BHT

Reviewer: Is this block the same of above? Can we merge the three of them into a single checklist that is applied to each task?

- [ ] Open each *MRIQC* report on a current Web Browser (*Google Chrome* is preferred).
- [ ] Repeat the following two steps for each echo:
    - [ ] Visualize the first mosaic (standard-deviation) and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#standard-deviation-of-signal-through-time)
    - [ ] Scroll down to the carpetplot and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#carpetplot-and-nuisance-signals).
- [ ] Once you went through all the echo-wise visualization of the base report, scroll down to the "Extended echo-wise reports" section, inspect the background view and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#view-of-the-background-of-the-voxel-wise-average-of-the-bold-timeseries).
- [ ] Scroll down to the average BOLD mosaic and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#average-signal-through-time)
- [ ] Inspect the zoomed-in view of the average BOLD mosaic as well and apply the same [exclusion criteria](qaqc-criteria-unprocessed.md#average-signal-through-time).

> #### RSfMRI

Reviewer: ditto

- [ ] Repeat the following two steps for each echo:
    - [ ] Visualize the first mosaic (standard-deviation) and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#standard-deviation-of-signal-through-time)
    - [ ] Scroll down to the carpetplot and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#carpetplot-and-nuisance-signals). 
- [ ] Once you went through all the echo-wise visualization of the base report, scroll down to the "Extended echo-wise reports" section, inspect the background view and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#view-of-the-background-of-the-voxel-wise-average-of-the-bold-timeseries).
- [ ] Scroll down to the average BOLD mosaic and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#average-signal-through-time)
- [ ] Inspect the zoomed-in view of the average BOLD mosaic as well and apply the same [exclusion criteria](qaqc-criteria-unprocessed.md#average-signal-through-time).

## Visualizing *MRIQC*'s group reports

### Assessing anatomical images
- [ ] Open the *MRIQC* group report on a current Web Browser (*Google Chrome* is preferred).

> - [ ] Visualize the IQMs distributions and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#group-report).

Reviewer: ```suggestion
- [ ] Visualize the IQM distributions and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#group-report).
```

### Assessing functional images
- [ ] Open the *MRIQC* group report on a current Web Browser (*Google Chrome* is preferred).

> - [ ] Visualize the IQMs distributions and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#group-report-1).

Reviewer: ```suggestion
- [ ] Visualize the IQM distributions and apply the [exclusion criteria](qaqc-criteria-unprocessed.md#group-report-1).
```
